src/internal/utils.ts

```diff
@@ -15,28 +15,26 @@
  */
 
 import log from 'loglevel';
+import platform from 'platform';
+
 import { CallServiceData, CallServiceResult, CallServiceResultType, ResultCodes } from './commonTypes';
 import { FluencePeer } from './FluencePeer';
-import { Particle, ParticleExecutionStage } from './Particle';
+import { ParticleExecutionStage } from './Particle';
 import Buffer from './Buffer';
-import platform from 'platform';
 
-export const MakeServiceCall = (fn: (args: any[]) => CallServiceResultType) => {
-    return (req: CallServiceData): CallServiceResult => {
-        return {
-            retCode: ResultCodes.success,
-            result: fn(req.args),
-        };
-    };
-};
+export const MakeServiceCall =
```

Comment:
To my taste the former is more explicit, thus easier to understand in a quick glance

Reply:
I always remove the syntax that is not required if it doesn't harm readability. If you sure you want this reversed I can do it. It's all purely a taste thing after all. We could even decide and have a eslint rule for that so we never discuss this again